package.json

```diff
@@ -33,7 +33,6 @@
     "storybook:build": "cross-env NODE_NO_WARNINGS=1 storybook build --quiet --webpack-stats-json",
     "test:unit": "cross-env NODE_NO_WARNINGS=1 jest",
     "test:unit:watch": "npm run test:unit -- --watch",
-    "test": "npm run test:unit",
```

Comment:
I couldn't find anywhere that this command was used so I removed it as a small cleanup. If it is indeed useful, happy to bring it back.

Reply:
Found it in the pull request template!